xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -901,7 +901,7 @@ For comments, this value points to before the first comment delimiter (that is,
         <Parameter Name="value" Type="System.Byte[]" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="value">When the method returns, a byte array that contains the base 64-encoded JSON string if the method succeeds.</param>
+        <param name="value">If the method succeeds, a byte array that contains the base 64-encoded JSON string.</param>
```

Comment:
Can we make the same change to the other doc comment, for `JsonElement.TryGetBytesFromBase64`? 
https://github.com/dotnet/dotnet-api-docs/pull/2737/files#diff-5b79fe7599d21bf8a1436d585fe965eaR755

Reply:
Maybe:
```suggestion
        <param name="value">If the method succeeds, contains the decoded binary representation of the base 64 text.</param>
```

Reply:
Sure, I'll add it manually.